### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by sean-m-slattery**

### CODE REVIEW

**Comment by divideandconquer** [comment]
looks good to me.  I'd like to see a review from someone who knows ruby better than i do.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,63 @@
...
+
+Add the following to your docker-compose.yml 's environment section for the app that will be using Parameter Store
+```      AWS_REGION: us-east-1
+      AWS_SECURITY_TOKEN: $AWS_SECURITY_TOKEN
+      AWS_ACCESS_KEY_ID: $AWS_ACCESS_KEY_ID
+      AWS_DEFAULT_REGION: us-east-1
+      AWS_SESSION_TOKEN: $AWS_SESSION_TOKEN
+      AWS_SECRET_ACCESS_KEY: $AWS_SECRET_ACCESS_KEY
+```
+To run the container, just add aws-vault exec pbxx-dev -- before your command
```
Comment (tsharkey) [inline README.md]: might just want to wrap this so people don't think the -- is not part of the command or something. I could see myself making that mistake haha. `aws-vault exec pbxx-dev --`

Comment (sean-m-slattery) [reply README.md]: ah good call

**Review by jhilde — APPROVED** [review]
Gave it a rails once over. Looks great! Excited to integrate into accounting-service.

**Pull request merged**